torch/ao/quantization/fuser_method_mappings.py

```diff
@@ -142,7 +136,6 @@ def fuse_convtranspose_bn(is_qat, convt, bn):
         "ConvTranspose and BN both must be in the same mode (train or eval)."
 
     if is_qat:
-        assert convt.training, "qat is only supported when convt.training is True currently"
         raise Exception("Fusing ConvTranspose+BatchNorm not yet supported in training.")
```

Comment:
we can update the exception message as well, we can say it does not support fusion for quantization aware training